containers/mlflow-tracking-server/Dockerfile

```diff
@@ -16,14 +16,16 @@
 
 FROM python:3.7.6
 
-ARG MLFLOW_VERSION=1.5.0
-ENV MLFLOW_RUNS_DIRECTORY=/mlruns
+ARG MLFLOW_VERSION=1.11.0
+ARG PSYCOPG2_VERSION=2.8.6
+ENV MLFLOW_ARTIFACT_ROOT=/mlruns 
+ENV MLFLOW_BACKEND_STORE=/mlruns
 
-RUN pip install --no-cache-dir mlflow==${MLFLOW_VERSION}
+RUN pip install --no-cache-dir mlflow==${MLFLOW_VERSION} psycopg2==${PSYCOPG2_VERSION}
 
-RUN mkdir -p ${MLFLOW_RUNS_DIRECTORY}
+RUN mkdir -p ${MLFLOW_ARTIFACT_ROOT}
 
 EXPOSE 5000
 
 ENTRYPOINT ["bash", "-c"]
-CMD ["mlflow server --backend-store-uri ${MLFLOW_RUNS_DIRECTORY} --host 0.0.0.0 -p 5000"]
+CMD ["mlflow server --backend-store-uri ${MLFLOW_BACKEND_STORE} --default-artifact-root ${MLFLOW_ARTIFACT_ROOT} --host 0.0.0.0 -p 5000"]
```

Comment:
it's from the ticket https://github.com/odahu/odahu-trainer/issues/21:

> supports in cluster PG deployment as database for MLFlow. Database in PG is already present

📔 **Note**
In order to use model registry functionality, you must run your server using a database-backed store.

**Question:** Are we going to store our models in **MLflow Registry** or am I missing something?